frameworks/elastic/versions.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-export TEMPLATE_ELASTIC_VERSION="7.6.0"
+export TEMPLATE_ELASTIC_VERSION="7.9.3"
```

Comment:
Can we bump Kibana to `7.9.3` also? After all, they are released together.

Reply:
Kibana version depends on this variable, so it is already updated
https://github.com/mesosphere/dcos-elastic-service/blob/master/frameworks/elastic/universe-kibana/resource.json#L4

Reply:
yeah! assuming the assets are already uploaded it looks good. :heavy_check_mark: